packages/components/src/mobile/alt-text-settings/styles.scss

```diff
@@ -0,0 +1,29 @@
+.altTextEditor {
+	padding: 14px 0;
+	min-height: 80px;
+	max-height: 250px;
+}
+
+.altTextEditorDark {
+	color: $white;
+}
+
+.altTextFootnote {
+	font-size: $default-font-size;
```

Comment:
From what I can tell, this is a web-defined variable, which I'm not sure is what we'd like here. I'm not if we've got the it implemented quite yet, but in Figma I've typically been using a "Caption" from this [neutral type-scale](https://zeroheight.com/226850145/p/197239-typography/b/24b504), which is 12px. 

Are we using this same variable on the other instances of the section footer component (columns, etc)? If so, let's just use what you have here (`$default-font-size`) and worry about the neutral type-scale at a later time.

Reply:
> Are we using this same variable on the other instances of the section footer component (columns, etc)? If so, let's just use what you have here ($default-font-size) and worry about the neutral type-scale at a later time.

[The `FooterMessageControl` component](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/footer-message-control/index.native.js) being used for the columns block uses a font-size of 15px (with the `$text-editor-font-size` variable) and the text is centred:

![Screenshot_1615444170](https://user-images.githubusercontent.com/2998162/110745588-62f37f00-8233-11eb-89a9-21064d15f999.png)

I found the `$default-font-size` variable in the same file where `$text-editor-font-size` ([`_variables.scss`](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_variables.scss)) is declared, which is how I ended up using it.

I've changed this to 12px now. :)

Reply:
Thank you! One thing I noticed relative to the screenshots in this other comment — it looks like the text color (not the link) in your PR is a tad darker than in the screenshot above (Columns settings). Can we align our footnote color here with the one on Columns?

Reply:
@iamthomasbishop, it looks like the colour contrast in the Columns setting is a bit below what's outlined in the accessibility standards for WordPress. I've gone ahead to open a separate issue for that in https://github.com/WordPress/gutenberg/issues/29866. I ended up using the `$gray-text-min` variable I found in [the _colors.native.scss file](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.native.scss#L50) to get the lightest colour that passes the standards here. Do you think that seems fair or is there perhaps another colour you'd prefer to use? I can submit a PR to also update the colour of the Footer Message component depending on what you think.